plugins/simpleadminhacks-paper/src/main/java/com/programmerdan/minecraft/simpleadminhacks/framework/utilities/BuildLimit.java

```diff
@@ -0,0 +1,81 @@
+package com.programmerdan.minecraft.simpleadminhacks.framework.utilities;
+
+import org.bukkit.configuration.serialization.ConfigurationSerializable;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class BuildLimit implements ConfigurationSerializable {
+	private String world;
+	private String type;
+	private int minY;
+	private int maxY;
+
+	public BuildLimit (String world, String type, int minY, int maxY){
+		this.world = world;
+		this.type = type;
+		this.minY = minY;
+		this.maxY = maxY;
+	}
+
+	public String getWorld() {
+		return this.world;
+	}
+	public String getType() {
+		return this.type;
+	}
+	public int getMinY() {
+		return this.minY;
+	}
+	public int getMaxY() {
+		return this.maxY;
+	}
+
+	public void setWorld(String world) {
```

Comment:
are the setters necessary?

Reply:
I think they are. There isn't a great lot of documentation on serialisations so I used what I could find

Reply:
Why?